examples/ecs-firelens/container-definitions-fargate.template.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "example",
+    "image": "<account-id>.dkr.ecr.<region>.amazonaws.com/<image-name>:latest",
+    "environment": [{
+      "name": "AWS_EMF_AGENT_ENDPOINT",
+      "value": "tcp://127.0.0.1:25888"
+    }
+    ],
+    "essential": true,
+    "logConfiguration": {
+      "logDriver": "awsfirelens",
+      "options": {
+        "Name": "cloudwatch",
+        "region": "<region>",
+        "log_key": "log",
+        "log_group_name": "aws-emf-ecs-firelens-example-metrics",
+        "auto_create_group": "true",
+        "log_stream_prefix": "emf-",
```

Comment:
In extra.config we set log_stream_prefix to be `log_stream_prefix   from-fluent-bit-`, in here it is `emf-`. Which one is prior?  also do we need "options" here since we already have `extra.conf` that define these fields

Reply:
Got it. Can we separate stdout log group with the log group the metrics logger will send to? I remembered that's what recommended by firelens team

Reply:
The recommendation from their side was to separate the logs that Fluent-bit writes vs the logs that our application container is emitting.

Reply:
>The recommendation from their side was to separate the logs that Fluent-bit writes vs the logs that our application container is emitting.

What does separate logs mean. separate log stream? separate log group?

Reply:
Separate log group only.